src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -236,15 +236,15 @@ static std::vector<SenderThreadParams*>* buildSenderParamsCustom
     {
       if (macroSubstitute(&payload, httpInfo.payload, ce) == false)
       {
-        LM_W(("Bad Input (not sending NotifyContextRequest: payload too large)"));
+        LM_W(("Runtime Error (not sending NotifyContextRequest: payload too large)"));
```

Comment:
Regarding 

> In the case of missing ending '}', it is not payload too large ...

You are right. However, instead of using an enum as return value (complex) a simpler approach is to move the log message to macroSubstitude().

Regarding 

> a missing ending '}' should not cause an error, simply not being replaced and kept as is ...

Although from a theoretical point of view a user could decice to print `${temperature` in its payload, the 99.99% cases this would corresond to human error situation in which the user missed `}`. Thus I think we should keep the current error detection in that case.